user-manual/administer/settings.rst

```diff
@@ -1871,3 +1872,95 @@ setting.
 
 
 :ref:`Back to top <settings>`
+
+.. _privacy-notification:
+
+Privacy notification
+====================
+
+.. NOTE::
+   This feature is new in the AtoM 2.4.1 release and is not present or available
+   in any previous versions (see `Release Notes 2.4.1 <https://wiki.accesstomemory.org/Releases/Release_announcements/Release_2.4.1>`)
```

Comment:
Don't forget to keep all your text wrapped to 80 characters, please!  This is outlined in point 10 of the following guidelines: https://wiki.accesstomemory.org/Resources/Documentation/Contribution_guidelines#Before_contributing

Remember, if you've installed the [magic-reflow](https://atom.io/packages/magic-reflow) package in the atom.io text editor, you can highlight a block of text and hit `ALT+Q` to automatically hard wrap text at 80 chars. I would suggest doing this in bits, so you can make sure it doesn't mess up the formatting of certain elements (like chopping a link mid hyphen, which can sometimes trigger sphinx errors, or messing up a bulleted list).